### PR TITLE
ggplot lesson revisions

### PR DESCRIPTION
**Pull request by stephhazlitt**
- minor text changes
- updates facet_wrap() to use `vars()`
- changes to `theme()` lesson so we are not to teaching ppl to change the angle of axis labels
- adds introduction to built in `themes_*()`
- some code re-formatting for style only

### CODE REVIEW

**Comment by stephhazlitt** [comment]
I tried to request you @gcperk to review as well but your handle did not auto-populate?

File: 06-plot-ggplot2.Rmd [hunk 1]
```diff
@@ -286,7 +297,8 @@ variables and their visual representation.
 > >
 > > ```{r ch4a-sol}
 > > ggplot(data = gapminder, mapping = aes(x = gdpPercap, y = lifeExp)) +
-> >  geom_point(size = 3, colour = "orange") + scale_x_log10() +
+> >  geom_point(size = 3, colour = "orange") +
+> > scale_x_log10() +
```
Comment (boshek) [inline 06-plot-ggplot2.Rmd:301]:
```suggestion
> >  scale_x_log10() +
```

File: 06-plot-ggplot2.Rmd [hunk 2]
```diff
@@ -333,12 +346,10 @@ asia_oceania <- gapminder[gapminder$continent == c("Asia", "Oceania"),]
...
 ggplot(data = asia_oceania, mapping = aes(x = year, y = lifeExp)) +
   geom_line() + 
-  facet_wrap( ~ country)
+  facet_wrap(vars(country))
 ```
 
-The `facet_wrap` layer took a "formula" as its argument, denoted by the tilde
-(~). This tells R to draw a panel for each unique value in the country column
-of the gapminder dataset.
+The `facet_wrap` layer took the function `vars()` as its argument. This tells R to draw a panel for each unique value in the variable country of the gapminder dataset.
```
Comment (boshek) [inline 06-plot-ggplot2.Rmd:352]:
```suggestion
Here we supplied the `vars` function to `facet_wrap`'s `facets` argument to easily specify which variable(s) we want to create panels for. In this case, we are telling R to draw a panel for each unique value in the variable country of the gapminder dataset.
```

File: 06-plot-ggplot2.Rmd [hunk 3]
```diff
@@ -356,22 +367,39 @@ of a fill legend would be set using `fill = "MyTitle"`.
...
   ) +
-  theme(axis.text.x = element_text(angle = 90, hjust = 1),
+  theme(axis.text.x = element_text(size = 5),
         plot.title = element_text(face = "bold"))
 ```
 
 > ## Tip
-> Note that we apply a "theme" definition to rotate
-> the x-axis labels to maintain readability.  Nearly everything in
+> Note that we apply a "theme" definition to customaize
```
Comment (boshek) [inline 06-plot-ggplot2.Rmd:383]:
```suggestion
> Note that we apply a "theme" definition to customize 
```

**Comment by stephhazlitt** [comment]
I did wonder as I committed all of those pngs, which is why I did them separately. There are some in the remote repo already so I gambled. Happy to roll back.

File: 06-plot-ggplot2.Rmd (hunk 2 above)
Comment (stephhazlitt) [reply 06-plot-ggplot2.Rmd:352]: Thanks for the the suggested edit, I think I am going to make a mash-up of this with the original commit.

File: 06-plot-ggplot2.Rmd [hunk 4]
```diff
@@ -356,22 +367,39 @@ of a fill legend would be set using `fill = "MyTitle"`.
...
+  theme(axis.text.x = element_text(size = 5),
         plot.title = element_text(face = "bold"))
 ```
 
 > ## Tip
-> Note that we apply a "theme" definition to rotate
-> the x-axis labels to maintain readability.  Nearly everything in
+> Note that we apply a "theme" definition to customize 
+> the x-axis label sizes to maintain readability.  Nearly everything in
 > ggplot2 is customizable.
```
Comment (ateucher) [inline 06-plot-ggplot2.Rmd:385]: Great addition showing the built-in themes. What do you think about a quick demo of `?theme` and `?ggtheme` to show the miriad of options and reinforce using `?` help?

Comment (stephhazlitt) [reply 06-plot-ggplot2.Rmd:385]: Yeah, I think that is a great idea. Do you think this needs to be captured in the bookdown or could it be an 'on the fly' element?

**Comment by stephhazlitt** [comment]
Thanks @gcperk, can you approve the PR?

**Pull request merged**